src/utils/user-utils.ts

```diff
@@ -0,0 +1,12 @@
+import type { User } from "src/generated/homeLambdasClient/models/User";
+import config from "src/app/config";
+
+/**
+ * To get the severa user id when logged in, if severaUserId is not found, it will return the test user severa id
+ *
+ * @param user
+ * @returns user severaUserId or testUserSeveraId
+ */
+export const getSeveraUserId = (user: User | undefined): string => {
+  return user?.attributes?.severaUserId ?? config.user.testUserSeveraId ?? "";
```

Comment:
This might need some thinking. We should not need this in production. As end users should have severaUserId. 
Do you mean that UI should give some message that user is using "test" data.

Reply:
I mean currently if there is no user.attributes.severaUserId then it falls back to the testUserId, but it should not do this in production, rather it should render some error message informing user they do not have a serveraId linked to their account.

This will likely need some changes once we have the servera optIn keyword resolved.

At minumum I'd do a check for the environement, this is fine in develop condition but in production condition there should be an error string rendered if user does not have a serveraUserId. Probably will need to add some condition that the balance card does not render at all unless the user has severa optIn keyword (but this can be done in a future feature).